pyproject.toml

```diff
@@ -1,3 +1,38 @@
+[project]
+name = "WhoCanIVoteFor"
+version = "0.0.0"
+description = "🗳 The source for https://whocanivotefor.co.uk/"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "django-filter==24.2",
+    "django==4.2.18",
+    "psycopg2-binary==2.9.9",
+    "requests==2.31.0",
+    "django-model-utils==4.3.1",
+    "markdown==3.5.1",
+    "django-localflavor==4.0",
+    "django-extensions==3.2.3",
+    "djangorestframework==3.14.0",
+    "icalendar==5.0.7",
+    "dc-django-utils",
+    "dc-logging-utils",
+    "djangorestframework-jsonp==1.0.2",
+    "feedparser==6.0.10",
+    "uk-election-timetables==4.1.0",
```

Comment:
Also need to bump this in line with https://github.com/DemocracyClub/WhoCanIVoteFor/pull/2183